### PR TITLE
Fixing bug : Check item type when found folder sequence

### PR DESCRIPTION
**Pull request by mguiral**
#61

### CODE REVIEW

File: test/pyTest/testStat.py [hunk 1]
```diff
@@ -30,6 +30,7 @@ def setUp():
 
     # create sym link
     os.symlink(os.path.join(root_path, "plop.txt"), os.path.join(root_path, "plop_sym_link.txt"))
+    os.symlink(os.path.join(root_path, "dir1"), os.path.join(root_path, "dir2"))
```
Comment (cchampet) [inline test/pyTest/testStat.py]: Could this line be removed?

File: test/pyTest/testLink.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    global root_path
+    shutil.rmtree(root_path)
+
+def testLinkType():
+    """
+    Check stats of a synbolic link folder.
+    - 2 hard links (. and ..)
+    - sizes
+    """
+    print("testFolderLinkStat:")
```
Comment (cchampet) [inline test/pyTest/testLink.py]: Could these print be removed?

File: test/pyTest/testLink.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    os.symlink(os.path.join(root_path, "plop.txt"), os.path.join(root_path, "plop_sym_link.txt"))
+    os.symlink(os.path.join(root_path, "dir1"), os.path.join(root_path, "dir2"))
+
+def tearDown():
+    global root_path
+    shutil.rmtree(root_path)
+
+def testLinkType():
+    """
+    Check stats of a synbolic link folder.
```
Comment (cchampet) [inline test/pyTest/testLink.py]: Could you update this doc? Something like:
"""
Check the type of the items found.
"""

File: test/pyTest/testLink.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    ]
+    for f in files_to_create:
+        createFile(root_path, f)
+
+    # create folder
+    createFolder(root_path, "dir1")
+
+    # create sym link
+    os.symlink(os.path.join(root_path, "plop.txt"), os.path.join(root_path, "plop_sym_link.txt"))
+    os.symlink(os.path.join(root_path, "dir1"), os.path.join(root_path, "dir2"))
```
Comment (cchampet) [inline test/pyTest/testLink.py:29]: What do you think to rename **dir2** to **dir1_sym_link**?

File: src/sequenceParser/filesystem.cpp [hunk 5]
```diff
@@ -154,7 +154,7 @@ std::vector<Item> browse(
 				// It's a sequence of directories, so it's not a sequence.
 				BOOST_FOREACH( Time t, s.getFramesIterable() )
 				{
-					output.push_back( Item( eTypeFolder, directory / s.getFilenameAt(t) ) );
+					output.push_back( Item( getTypeFromPath(directory / s.getFilenameAt(t)), directory / s.getFilenameAt(t) ) );
```
Comment (fabiencastan) [inline src/sequenceParser/filesystem.cpp]: You can create a variable for `directory / s.getFilenameAt(t)`.

File: test/pyTest/testStat.py (hunk 1 above)
Comment (mguiral) [reply test/pyTest/testStat.py]: Yes i will remove it

File: test/pyTest/testLink.py (hunk 2 above)
Comment (mguiral) [reply test/pyTest/testLink.py]: Yes i will remove it

File: test/pyTest/testLink.py (hunk 4 above)
Comment (mguiral) [reply test/pyTest/testLink.py:29]: The aim is to test the browse method when folder sequence detected

File: src/sequenceParser/filesystem.cpp [hunk 6]
```diff
@@ -154,14 +154,16 @@ std::vector<Item> browse(
 				// It's a sequence of directories, so it's not a sequence.
 				BOOST_FOREACH( Time t, s.getFramesIterable() )
 				{
-					output.push_back( Item( getTypeFromPath(directory / s.getFilenameAt(t)), directory / s.getFilenameAt(t) ) );
+          boost::filesystem::path folderPath = directory / s.getFilenameAt(t);
```
Comment (cchampet) [inline src/sequenceParser/filesystem.cpp]: :warning: indentation  :)

File: test/pyTest/testLink.py (hunk 4 above)
Comment (cchampet) [reply test/pyTest/testLink.py:29]: Oups yes you are right: so the name **dir2** is important.

**Pull request merged**